android/res/layout/fragment_color_grid.xml

```diff
@@ -5,4 +5,5 @@
     android:layout_height="wrap_content"
     android:paddingTop="@dimen/margin_half"
     android:gravity="center"
+    android:layoutDirection="ltr"
```

Comment:
That is a bad UX to force the LTR layout for RTL users. What causes the issue in the first place? Why this line fixes it for RTL? Will forcing `rtl` here fix it?

Reply:
I have check when we use language with RTL, round color is present on pop-up but color is not shown.
<img src="https://user-images.githubusercontent.com/87148630/227491788-b716face-c320-4ea0-a6ef-7f4a63a52e62.png" height=100 />

Reply:
> That is a bad UX to force the LTR layout for RTL users. What causes the issue in the first place? Why this line fixes it for RTL? Will forcing `rtl` here fix it?

RTL is disabled for the grid only, all other parts are still OK in RTL. Please see screenshots.

Reply:
I saw screenshots, and wrote the "bad UX" commend explicitly about non-wrapped bookmarks.

What caused this bug? Why this hacky fix fixes it?

Reply:
The grid looks symmetrical. All captions are correctly right-to-left. Sorry, I don't understand what is wrong here.

Reply:
Red starred item should be in the top right, not in the top left.

What caused this bug? Why this hacky fix fixes it?

Reply:
Actually in RTL red is at right.
Maybe it's work like drawable, Android reverse item but in your case, color is just in one face of item and other face don't have color

Reply:
Why does the order of colors matters here?

Reply:
The order of colors is not critical. But it was critical to understand the source of the problem.
It looks like a Google issue that they didn't fix since Android 4: https://issuetracker.google.com/issues/36982185

So your workaround is the only possible one we can use. No other options.

Can you please add a comment to the XML to clarify that we're forcing ltr here because of that google issue?